src/app/pages/HomePage/index.test.tsx

```diff
@@ -127,29 +136,48 @@ describe('Home Page', () => {
   });
 
   describe('Lazy Loading', () => {
-    it('Only the first image and message banner on the homepage are not lazy loaded, but all others are', () => {
-      render(<HomePage pageData={homePageData} />, {
-        service: 'kyrgyz',
-      });
+    test.each([[homePageData], [homePageDataWithBillboards]])(
+      'Only the first image, message banner(s) and billboard(s) on the homepage are not lazy loaded, but all others are',
+      data => {
+        render(<HomePage pageData={data} />, {
+          service: 'kyrgyz',
+        });
 
-      const messageBannerImages: string[] = [];
-      document
-        .querySelectorAll(`[data-testid^="message-banner"] img`)
-        .forEach(image =>
-          messageBannerImages.push(image.getAttribute(`src`) || ''),
-        );
+        const messageBannerImages: string[] = [];
+        document
+          .querySelectorAll(`[data-testid^="message-banner"] img`)
+          .forEach(image =>
+            messageBannerImages.push(image.getAttribute(`src`) || ''),
+          );
 
-      const imageList = document.querySelectorAll('img');
-      imageList.forEach((image, index) => {
-        const src = image.getAttribute('src') || '';
+        const billboardImages: string[] = [];
+        document
+          .querySelectorAll(`[data-testid^="billboard"] img`)
+          .forEach(image =>
+            billboardImages.push(image.getAttribute(`src`) || ''),
+          );
```

Comment:
We should be able to combine both message banner & billboard Xpath into a single query selector using a comma - see https://developer.mozilla.org/en-US/docs/Web/API/Document/querySelectorAll

```
document.querySelectorAll(`[data-testid^="billboard"] img, [data-testid^="message-banner"] img`)
```

Reply:
I've refactored this locally but not pushed yet, cheers for the suggestion